gunicorn.conf

```diff
@@ -3,3 +3,4 @@ errorlog = "/app/logs/gunicorn_error.log"
 
 bind = '0.0.0.0:5000'
 workers = 4
+reload = True
```

Comment:
Isn't this more for dev: https://docs.gunicorn.org/en/stable/settings.html#reload